meta/main.yml

```diff
@@ -11,5 +11,5 @@ galaxy_info:
     - trusty
   categories:
   - development
-dependencies:
-  - { role: "azavea.java" }
+#dependencies:
```

Comment:
Are you installing `java6-runtime-headles` or equivalent on your own? The role fails to provision without it.

Reply:
@hectcastro would you merge the PR, if undo this disabled java dependency?
ps:  Unfort. my issue (https://github.com/ansible/ansible/issues/13215) didn't get enough traction .. at least there's more recent similar requests that would also help, like https://github.com/ansible/ansible/issues/34566

Reply:
I've now done as I proposed -^ in a new PR, AND also properly documenting the 3 new role vars
-> https://github.com/azavea/ansible-scala/pull/4 👍